bootstrap

```diff
@@ -34,18 +34,18 @@ elif [ -f /etc/redhat-release ]; then
     fi
 fi
 
-virtualenv --no-site-packages --distribute virtualenv
+virtualenv --no-site-packages --distribute virtualenv -p /usr/bin/python3
 
 # avoid pip bugs
 ./virtualenv/bin/pip install --upgrade pip
 
 # slightly old version of setuptools; newer fails w/ requests 0.14.0
-./virtualenv/bin/pip install setuptools==32.3.1
+#./virtualenv/bin/pip install setuptools==32.3.1
 
 ./virtualenv/bin/pip install -r requirements.txt
 
 # forbid setuptools from using the network because it'll try to use
 # easy_install, and we really wanted pip; next line will fail if pip
 # requirements.txt does not match setup.py requirements -- sucky but
 # good enough for now
-./virtualenv/bin/python setup.py develop
+./virtualenv/bin/python3 setup.py develop
```

Comment:
there is no need to provide `python3` here, since `./virtualenv/bin/python` is already pointed to python3 created in with `virtualenv`.